test/simulations/run_plcr_voting.js

```diff
@@ -0,0 +1,182 @@
+/* eslint-env mocha */
```

Comment:
probably don't want this to be included with the PR, but I'm not sure what to do with it. Wonder if we should set these up in another repo so we can continue to use scripts like this for development. Maybe include it in this commit, then we'll clean it up later.